<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jarvisUI</class>
 <widget class="QMainWindow" name="jarvisUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1361</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-320</x>
      <y>-130</y>
      <width>1911</width>
      <height>1071</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bg_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 15, 15);
background-color: rgb(112, 8, 1);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bg_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>851</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../G.U.I Material-20221121T175203Z-001/G.U.I Material/B.G/redbg.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bg_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 15, 15);
background-color: rgb(112, 8, 1);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gif_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../G.U.I Material-20221121T175203Z-001/G.U.I Material/ExtraGui/live.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gif_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>461</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../G.U.I Material-20221121T175203Z-001/G.U.I Material/ExtraGui/Code_Template.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gif_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../G.U.I Material-20221121T175203Z-001/G.U.I Material/B.G/Iron_Template_1.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_time">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
border-radius:none;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg_5">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>510</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../G.U.I Material-20221121T175203Z-001/G.U.I Material/Buttons/button.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bg_6">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>510</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../G.U.I Material-20221121T175203Z-001/G.U.I Material/Buttons/button.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="bg_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>510</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../G.U.I Material-20221121T175203Z-001/G.U.I Material/Buttons/button.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_day">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>520</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
border-radius:none;
</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_date">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>520</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Transparent;
border-radius:none;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(225, 221, 217);
font: 75 22pt &quot;Courier New&quot;;
border-radius:none;</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 15, 15);
background-color: rgb(112, 8, 1);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_exit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>680</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(225, 221, 217);
font: 75 22pt &quot;Courier New&quot;;
border-radius:none;</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="bg_9">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>590</y>
      <width>621</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 15, 15);
color: rgb(230, 255, 250);
font: 75 18pt &quot;Courier New&quot;;
background-color: rgb(112, 8, 1);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_exit_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>600</y>
      <width>631</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(225, 221, 217);
font: 75 18pt &quot;Courier New&quot;;
border-radius:none;</string>
    </property>
    <property name="text">
     <string>J.A.R.V.I.S - A Voice Assistant</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>bg_8</zorder>
   <zorder>bg_2</zorder>
   <zorder>bg_4</zorder>
   <zorder>gif_2</zorder>
   <zorder>gif_3</zorder>
   <zorder>bg_3</zorder>
   <zorder>gif_1</zorder>
   <zorder>text_time</zorder>
   <zorder>bg_5</zorder>
   <zorder>bg_6</zorder>
   <zorder>bg_7</zorder>
   <zorder>text_day</zorder>
   <zorder>text_date</zorder>
   <zorder>push_start</zorder>
   <zorder>push_exit</zorder>
   <zorder>bg_9</zorder>
   <zorder>push_exit_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
